src/views/Home.vue

```diff
@@ -1,150 +1,102 @@
 <template>
-  <div class="home">
-    <div class="container">
-    <h4>Choose category</h4>
-    <select name="" id="" v-model="curCategory">
-      <option value="">Choose</option>
-      <option v-for="opt in categories" v-bind:value="opt.value" v-bind:key="opt.value">
-        {{ opt.text }}
-      </option>
-    </select>
-      <div class="row" v-if="!finished && curCategory">
-        <h2>Gunluk pratik</h2>
-        <div class="col-8">
-          <div class="card mb-3">
-            <div class="card-body">
-              <p class="card-text">{{ questions[curCategory][index[curCategory]].q }}</p>
-              <p v-if="isRevealed">{{ questions[curCategory][index[curCategory]].a }}</p>
-              <button class="btn btn-primary" @click="reveal" v-if="!isRevealed">Goster</button>
-              <button class="btn btn-success" v-show="isRevealed"
-              @click="remembered">Hatirladim</button>
-              <button class="btn btn-danger" v-show="isRevealed"
-              @click="notRemembered">Hatirlayamadim</button>
+<div class="home">
+    <div v-if="!currentUser"><br><br><br></div>
```

Comment:
no `<br>` to style, use proper class names like `mt-1, mb-2, ... mr-5` from bootstrap